ui/src/components/HelpDrawer.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from "react";
+import styled from "styled-components";
+import Drawer from "@material-ui/core/Drawer";
+import Grid from "@material-ui/core/Grid";
+import CloseIcon from "@material-ui/icons/Close";
+import { HelpAccordian } from "./HelpAccordian";
+import { colors } from "../styles/colors";
+import {
+  HelpDrawerLaunchButtonText,
+  HelpDrawerBoxText,
+  HelpDrawerTitleText,
+} from "../styles/typography";
+import { TutorialState } from "../components/WelcomeTutorial";
+
+const StyledRoot = styled.div`
+  width: 25vw;
+  height: 100vh;
+  margin: 24px;
+  position: relative;
+`;
+
+const ScrollArea = styled.div`
+  height: 78vh;
+  overflow: scroll;
+`;
+
+const StyledBox = styled.div`
+  border: 1.5px solid ${colors.primary};
+  background: ${colors.primaryLight4}80;
+  border-radius: 10px;
+  margin: 12px 0px;
+  padding: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+`;
+
+const CloseButton = styled.button`
+  width: 24px;
+  height: 24px;
+  border: none;
+  background: none;
+  padding: 0;
+  color: ${colors.primaryDark1};
+
+  &:hover {
+    opacity: 70%;
+    cursor: pointer;
+  }
+
+  &:focus {
+    outline: 0;
+    border-radius: 50%;
+    background-color: ${colors.neutral};
+    cursor: pointer;
+  }
+`;
+
+const LaunchButton = styled.button`
+  width: 100%;
+  height: 48px;
+  margin: 24px 24px 0 24px;
+  border-radius: 5px;
+  border: none;
+  background-color: ${colors.primaryLight4};
+  cursor: pointer;
+  z-index: 100;
+  position: absolute;
+  bottom: 0;
+
+  &:hover {
+    cursor: pointer;
+    background-color: ${colors.primaryLight3};
+  }
+
+  &:focus {
+    cursor: pointer;
+    background-color: ${colors.primaryLight2};
+  }
+`;
+
+export enum HelpDrawerState {
+  Open,
+  Closed,
+}
+
+interface HelpDrawerProps {
+  state: HelpDrawerState;
+  setIsHelpDrawerOpen: (newState: HelpDrawerState) => void;
+  setIsTutorialOpen: (newState: TutorialState) => void;
+}
+
+export function HelpDrawer({
+  state,
+  setIsHelpDrawerOpen,
+  setIsTutorialOpen,
+}: HelpDrawerProps): JSX.Element {
+  const handleLaunchTour = () => {
+    setIsTutorialOpen(TutorialState.First);
+    setIsHelpDrawerOpen(HelpDrawerState.Closed);
+  };
+
```

Comment:
I think the zoom controls need to move left when to still be visible when the panel is open?
![image](https://user-images.githubusercontent.com/19617248/101398203-b86a6980-389b-11eb-8d88-64f9e932c65b.png)

Reply:
Ooh yes I forgot about that!